src/common/errorLogger.ts

```diff
@@ -1,11 +1,13 @@
-export function captureException(exception, options) {
+
+
+export function captureException(exception: Error, options?: { extra: Record<string, unknown> }) {
   if (process.env.NODE_ENV === 'production') {
     console.error('An error occured and was sent to Sentry.', exception);
     window.Sentry && window.Sentry.withScope(scope => {
       if (options && options.extra) {
         scope.setExtras(options.extra);
       }
-      window.Sentry.captureException(exception);
+      window.Sentry && window.Sentry.captureException(exception);
```

Comment:
TS has a shorthand for this:
```suggestion
      window.Sentry?.captureException(exception);
```